### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

before the upcoming updates to the license, these slight improvements dropped in while getting accustomed with the code base. A short summary:

- Improve `DEVELOP.rst` file for a quick ramp-up using CrateDB within Docker.
- Replace Travis CI with GitHub Actions. With caching of artifacts, it will only take 45s in total now.
- Make French translation work again.
- Fix minor whitespace issue.

With kind regards,
Andreas.

### CODE REVIEW

File: DEVELOP.rst [hunk 1]
```diff
@@ -41,11 +41,18 @@ Running the App for Development
...
 
-    $ env/bin/npm start
+    npm start
 
 You should now be able to access the app at: http://localhost:9000/
 
 
+Running CrateDB
+===============
+The most quick way to spin up an instance is probably to use Docker::
```
Comment (matthijskrul) [inline DEVELOP.rst:51]: Quickest

Comment (amotl) [reply DEVELOP.rst:51]: Thanks! Sorry about this quirk.

Comment (seut) [inline DEVELOP.rst:51]: I think there should be an empty line after each section title.

**Review by seut — APPROVED** [review]
Lgtm otherwise

**Review by seut — APPROVED** [review]

**Pull request merged**